shared/team-building/container.tsx

```diff
@@ -680,6 +682,10 @@ const mergeProps = (
         : ownProps.decHighlightIndex,
     recommendations: recommendationsSections,
     recommendedHideYourself: ownProps.recommendedHideYourself,
+    refreshBlockList: () =>
+      ownProps.namespace === 'people' && searchResults
+        ? dispatchProps._refreshBlockList(searchResults.map(r => r.services['keybase'] || '').filter(Boolean))
```

Comment:
The `|| ''` part doesn't seem necessary

Reply:
Unfortunately it is necessary since otherwise Typescript doesn't realize that the filter changes `string?` to `string`. But anyway I'm going to refactor this based on your suggestion below, so it might end up being irrelevant.

Reply:
Ah, bummer!